src/Microsoft.ML.Parquet/ParquetLoader.cs

```diff
@@ -678,17 +678,17 @@ public ParquetConversions(IChannel channel)
 
             public void Conv(ref byte[] src, ref VBuffer<Byte> dst) => dst = src != null ? new VBuffer<byte>(src.Length, src) : new VBuffer<byte>(0, new byte[0]);
 
-            public void Conv(ref sbyte? src, ref DvInt1 dst) => dst = src ?? DvInt1.NA;
+            public void Conv(ref sbyte? src, ref SByte? dst) => dst = src;
 
             public void Conv(ref byte src, ref byte dst) => dst = src;
 
-            public void Conv(ref short? src, ref DvInt2 dst) => dst = src ?? DvInt2.NA;
+            public void Conv(ref short? src, ref Int16? dst) => dst = src;
 
             public void Conv(ref ushort src, ref ushort dst) => dst = src;
 
-            public void Conv(ref int? src, ref DvInt4 dst) => dst = src ?? DvInt4.NA;
+            public void Conv(ref int? src, ref Int32? dst) => dst = src;
 
-            public void Conv(ref long? src, ref DvInt8 dst) => dst = src ?? DvInt8.NA;
+            public void Conv(ref long? src, ref Int64? dst) => dst = src;
 
```

Comment:
I see there are no conversions from `float` to `float`, or `double` to `double`. So: are these conversions actually necessary?